.github/workflows/ci.yml

```diff
@@ -4,42 +4,41 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node:
           - 18
           - 20
-        react: [16, 17]
+        react: [16, 17, 18]
 
     name: build (${{ matrix.node }} w/ React ${{ matrix.react }}
 
     steps:
-    - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
 
-    - name: Use Node.js ${{ matrix.node }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ matrix.node }}
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node }}
 
-    - name: Install dependencies
-      run: npm ci
+      - name: Install dependencies
+        run: npm ci
 
-    - name: Install React v${{ matrix.react }}
-      run: npm i react@${{ matrix.react }} react-dom@${{ matrix.react }}
+      - name: Install React v${{ matrix.react }}
+        run: npm i react@${{ matrix.react }} react-dom@${{ matrix.react }}
 
-    - name: Install React 16 Enzyme adapter
-      if: matrix.react == '16'
-      run: |
-        npm remove @wojtekmaj/enzyme-adapter-react-17
-        npm install enzyme-adapter-react-16
+      - name: Install React 16 Enzyme adapter
+        if: matrix.react == '16'
+        run: |
+          npm remove @wojtekmaj/enzyme-adapter-react-17
+          npm install enzyme-adapter-react-16
```

Comment:
god, i forgot we're still using enzyme here

Reply:
ooops

https://github.com/readmeio/syntax-highlighter/pull/520/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R41-R42